routes/results/results.controller.js

```diff
@@ -1,17 +1,23 @@
 const { routeUtils, getSessionData } = require('./../../utils')
 const { Schema } = require('./schema.js')
-const { getBenefits } = require('./getBenefits');
+const { getBenefits } = require('./getBenefits')
+const _ = require('lodash')
 
 const getData = (req) => {
   /**
    * If there's querystring data use it,
    * otherwise get it from the session.
    */
-  if (req.query) {
-    return req.query;
+  console.log(`queryString ${JSON.stringify(req.query)}`)
```

Comment:
Did you mean to leave this console.log in place?

Reply:
No

Reply:
removed